packages/kbn-optimizer/limits.yml

```diff
@@ -54,7 +54,7 @@ pageLoadAssetSize:
   mapsLegacy: 116817
   mapsLegacyLicensing: 20214
   ml: 82187
-  monitoring: 268612
+  monitoring: 288612
```

Comment:
Our unofficial goal is to ultimately limit all page load asset sizes to 200kb in the next year or so, as teams continue to work reducing the size of their bundles. Is anyone from the monitoring team working to reduce the page load asset size of this bundle? a 20kb increase feels reasonable for now, but I'd like to ask that someone from the monitoring team take a look at the `x-pack/plugins/monitoring/target/public/stats.json` in one of the many [webpack analyzers](http://webpack.github.io/analyse/) or [visualizers](https://survivejs.com/webpack/optimizing/build-analysis/) after running `node scripts/build_kibana_platform_plugins --focus monitoring --profile`?

When I run it I see that the server code is being bundles in the page load bundle, which seems like the best way to fix the limit issue, rather than raising the limit.

![image](https://user-images.githubusercontent.com/1329312/97235047-47fe0200-179f-11eb-9f32-21ba3496c263.png)

> PS, I'm working on unifying the docs for the best way to diagnose and deal with this stuff now

Reply:
This is awesome! Thank you @spalger 🙇